README.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+This project implemented protoc plugin for generation services for gRPC router.
+It generates java and python code.
+You can plug it to protoc or run from command line.
+# Usage
+## Protoc plugin
+If you want to use this generator with gradle and protoc you should add artifact name to protoc plugins section and add to generated protoc tasks
+You can set options:
+1. javaInterfacesPath - generate services interfaces to this path
+1. javaInterfacesImplPath - generate TH2 services interfaces implementation to this path
+1. pythonPath - generate python services to this path
+
+All these paths will be relative from  ``generateProtoTasks.generatedFilesBaseDir`` (Example: `javaInterfacesPath` = `path/to/base/dir/path/to/interfaces`)
```

Comment:
Also, please add information about the default behaviour (when neither of those options is set)

```suggestion
All these paths are relative to the directory in  ``generateProtoTasks.generatedFilesBaseDir`` parameter (Example: `javaInterfacesPath` = `path/to/base/dir/path/to/interfaces`)
```